app/models/athlete.rb

```diff
@@ -0,0 +1,11 @@
+class Athlete < ApplicationRecord
+  validates_presence_of :name, :gender, :birthday
+  validates_numericality_of :height, :weight
```

Comment:
I had to look it up, but validates_numericality_of does not inherently check for the presence_of.

So you will want to add :height and :weight to validates_presence_of